package.json

```diff
@@ -45,6 +45,7 @@
     "singleQuote": true
   },
   "devDependencies": {
+    "@simbathesailor/use-what-changed": "^1.0.0",
```

Comment:
this seems unused

Reply:
I used it for debugging the `useEffect`, it could be useful for future debugging processes

Reply:
can you add something about it in the readme then? maybe with a link to the documentation?
otherwise nobody except you know why it's in the dependencies

Reply:
thanks